packages/browser/src/login/oidc/redirectHandler/AuthCodeRedirectHandler.ts

```diff
@@ -147,7 +147,7 @@ export class AuthCodeRedirectHandler implements IRedirectHandler {
         this.issuerConfigFetcher,
         this.clientRegistrar,
         // the canHandle function checks that the code is part of the query strings
-        url.query["code"] as string,
+        url.searchParams.get("code") as string,
```

Comment:
ditto

Reply:
Not possible to add type guards on a Map lookup.